src/components/input/input-text/input-text.tsx

```diff
@@ -0,0 +1,57 @@
+import { useTextField } from "@react-aria/textfield";
+import React from "react";
+
+import classes from "./input-text.module.css";
+
+type InputTextProps = {
+  type: string;
+  name: string;
+  label: string;
+  autoFocus?: boolean;
+  onChange: (value: string) => void;
+  value?: string;
+  placeholder?: string;
+  className?: string;
+  maxLength?: number;
+  children?: React.ReactNode;
+  required?: boolean;
+  hasError?: boolean;
```

Comment:
A little proposal here.

Since we may display an error under pretty much every `InputText`, what do you think a adding here the error display:
```ts
        {errorMessage && (
          <FormErrorMessage>{errorMessage}</FormErrorMessage>
        )}
```
So when we import the input component, we don't have to repeat the same process for errors on each one of them.

Reply:
I see your point but, what about this one ? Putting this error message below the first input would be strange, no ? 🤔 
<img width="351" alt="image" src="https://user-images.githubusercontent.com/50053259/130621633-2ad898a1-1c66-4041-8106-2fd0778d541a.png">

Reply:
I don't think so because here, the error is about the first input.
The possible error for the 2nd one should only be "doesn't match the previous password" or smth like that.

Reply:
> I don't think so because here, the error is about the first input.

Well, I forgot about this one, but I'm really not a fan of this. There would be too much input jump :/.

Reply:
The thing is that we have two ways of displaying errors in forms right now. In Profile forms, we display them below the concerned input, and on the rest of the app, we display them above the form. I thought that what you were proposing here was to put the error message in `InputText` AND display it as we do for Profile, below the input. (If we want to have inside `InputText, we have to chose if we put it below or above). Anyway, I think having this long error message inserted between the two inputs feels strange to me.

Reply:
As I said, it's a proposal, do it as you prefer :)

Reply:
I'm gonna make a task about it, so we could discuss it on the next refinement 😉